### PR TITLE
deleter.java: added kaushik name

### PR DESCRIPTION
**Pull request by kaushiknath98**
Added //kaushik in deleter.java. Kindly review and merge.

### CODE REVIEW

File: src/com/corps/reddragons/Deleter.java [hunk 1]
```diff
@@ -4,4 +4,6 @@
 	
 	//mah life
 
+	//kaushik
```
Comment (jesusNut) [inline src/com/corps/reddragons/Deleter.java:7]: This is wrong. Please add this line at litte below.

**Review by jesusNut — CHANGES_REQUESTED** [review]

File: src/com/corps/reddragons/Deleter.java (hunk 1 above)
Comment (kaushiknath98) [reply src/com/corps/reddragons/Deleter.java:7]: this is right. Pleaase approve

Comment (jesusNut) [reply src/com/corps/reddragons/Deleter.java:7]: No..this is not right. I will fire you...

Comment (jesusNut) [reply src/com/corps/reddragons/Deleter.java:7]: Please do here the chnage.

**Review by jesusNut — CHANGES_REQUESTED** [review]

File: src/com/corps/reddragons/Deleter.java (hunk 1 above)
Comment (kaushiknath98) [reply src/com/corps/reddragons/Deleter.java:7]: go to hell..i will join Flipkart.

Comment (kaushiknath98) [reply src/com/corps/reddragons/Deleter.java:7]: Kaushik review

File: src/com/corps/reddragons/Deleter.java [hunk 2]
```diff
@@ -4,4 +4,8 @@
 	
 	//mah life
 
+	
+
```
Comment (jesusNut) [inline src/com/corps/reddragons/Deleter.java:8]: This looks good. Merging now.

**Review by jesusNut — APPROVED** [review]
All good. Merging with msater.

**Pull request merged**